docs/docs/erc20_onboarding.md

```diff
@@ -4,96 +4,161 @@ title: Onboarding ERC20 tokens to ZeppelinOS
 sidebar_label: ERC20 opt-in onboarding
 ---
 
-This guide describes how a legacy ERC20 token can be migrated to an upgradeable ERC20 through ZeppelinOS. 
-This proposal is based on the [opt-in onboarding strategy](https://github.com/zeppelinos/labs/tree/master/migrating_legacy_token_opt_in)
-we've been exploring. 
+## Intro
+This guide covers the migration of a regular ERC20 token to an upgradeable version of itself. During this process, the 
+original contract (to be called "legacy") and the new contract, which will have the ability of being upgradeable, will coexist.
+ 
+The _new upgradable contract_ will have the same functionality provided by the _legacy_ contract, but it will be 
+*upgradable*. This means that we will be able to add new functionality, store new data, fix bugs or support new 
+standards, without any need to perform new migrations in the future.  
```

Comment:
Emphasise that we will be able to add new functionality (etc) _any time we need_, or _as many times as we need_